csharp/BYOAuthFlow/Program.cs

```diff
@@ -0,0 +1,225 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using Google.Protobuf;
+using Google.Protobuf.WellKnownTypes;
+using Improbable.SpatialOS.Deployment.V1Alpha1;
+using Improbable.SpatialOS.Platform.Common;
+using Improbable.SpatialOS.PlayerAuth.V2Alpha1;
+using Improbable.Worker;
+using Improbable.Worker.Alpha;
+using Utils;
+using Deployment = Improbable.SpatialOS.Deployment.V1Alpha1.Deployment;
+using Locator = Improbable.Worker.Alpha.Locator;
+using LocatorParameters = Improbable.Worker.Alpha.LocatorParameters;
+
+namespace BYOAuthFlow
+{
+    internal class Program
+    {
+        /// <summary>
+        ///     PlEASE REPLACE ME.
+        ///     Your SpatialOS project name.
+        ///     It should be the same as the name specified in the local spaitalos.json file used to start spatiald.
+        /// </summary>
+        private const string ProjectName = "platform_sdk_examples";
+
+        /// <summary>
+        ///     PlEASE REPLACE ME.
+        ///     The path to a valid launch configuration json file.
+        /// </summary>
+        private const string LaunchConfigFilePath = "../../blank_project/default_launch.json";
+
+        /// <summary>
+        ///     PlEASE REPLACE ME.
+        ///     The assembly you would want the cloud deployment to use.
+        /// </summary>
+        private const string AssemblyId = "blank_project";
+
+        /// <summary>
+        /// The Spatiald port to use for the local scenario
+        /// </summary>
+        private const int SpatialdPort = 9876;
+
+
+        /// <summary>
+        ///     PlEASE REPLACE ME.
+        ///     The SpatialOS Platform refresh token of a service account or a user account.
+        /// </summary>
+        private static string RefreshToken =>
+            Environment.GetEnvironmentVariable("IMPROBABLE_REFRESH_TOKEN") ?? "5b6c50ed-f488-45f2-b02b-e6caa346b26e";
```

Comment:
where does this come from?

Reply:
`Error: fetch: server error: rpc error: code = NotFound desc = Unable to fetch refresh token 5b6c50ed-f488-45f2-b02b-e6caa346b26e`

Reply:
removed.